azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/Application.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.batch;
+
+import com.microsoft.azure.management.apigeneration.LangDefinition;
+import com.microsoft.azure.management.batch.implementation.ApplicationInner;
+import com.microsoft.azure.management.batch.implementation.ApplicationPackageInner;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ExternalChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Attachable;
+import com.microsoft.azure.management.resources.fluentcore.model.Settable;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+
+import java.util.List;
+
+/**
+ * An immutable client-side representation of an Azure batch account application.
+ */
+@LangDefinition(ContainerName = "~/")
+public interface Application extends
+        ExternalChildResource<Application>,
+        Wrapper<ApplicationInner> {
+
+    /**
+     * Get the displayName value.
```

Comment:
i suspect this Javadoc was copied from the auto-gen'd library.  We want to be more descriptive than that (but not too much either). This could simply say: "the application's display name"